### PR TITLE
Fixed Syntax Errors

### PR DESCRIPTION
**Pull request by Kriyszig**
Fixed syntax errors in documents caused by the map function not have a closing bracket. 
Also fixed an incorrect comment.

### CODE REVIEW

File: documentation/0045-node-folders/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ const isFile = fileName => {
 
 fs.readdirSync(folderPath).map(fileName => {
   return path.join(folderPath, fileName)).filter(isFile)
-}
+})
```
Comment (ghost) [inline documentation/0045-node-folders/index.md:63]: This isn't right. The map closes after the path.join() function but the `}` is definitely out of place.

**Review by ghost — CHANGES_REQUESTED** [review]

File: documentation/0045-node-folders/index.md (hunk 1 above)
Comment (Kriyszig) [reply documentation/0045-node-folders/index.md:63]: Fixed in cec8a13.
Thanks a lot for pointing that out.

**Review by ghost — APPROVED** [review]
LGTM

**Comment by ghost** [comment]
Thanks @VisibleMarkov for fixing this (^_^)

**Comment by ghost** [comment]
@amiller-gh,
Can we land this now given these errors exist in master?

**Review by ZanMarolt — APPROVED** [review]
Nice catch

**Comment by amiller-gh** [comment]
LGTM! Thanks for the catch!

**Pull request merged**